verified-inference/enclave/run.sh

```diff
@@ -7,8 +7,6 @@ ip link set dev lo up
 
 # Add a hosts record, pointing target site calls to local loopback
 echo "127.0.0.1   api.openai.com" >> /etc/hosts
-echo "127.0.0.1   api.mainnet-beta.solana.com" >> /etc/hosts
```

Comment:
why is this removed?

Reply:
It's because we no longer need access solana from TEE any more 
(not relevant to this PR, some historical leftovers)